components/Builder/Builder.tsx

```diff
@@ -1,20 +1,24 @@
-import { Tabs } from "components";
 import { SectionHeader } from "components/SectionHeader/SectionHeader";
 import { BaseOuterWrapper } from "components/style/Wrappers";
 import { laptopAndUnder, mobileAndUnder, tabletAndUnder } from "constant";
+import { useInView } from "framer-motion";
 import { useAddHashToUrl } from "hooks/helpers/useAddHashToUrl";
+import dynamic from "next/dynamic";
 import OO from "public/assets/oo-logo.svg";
 import { useRef } from "react";
 import styled from "styled-components";
 
+const Tabs = dynamic(() => import("components/Tabs/Tabs"));
+
 export function Builder() {
   const id = "builder";
   const ref = useRef<HTMLDivElement>(null);
+  const inView = useInView(ref, { amount: "some" });
```

Comment:
we only want to load these bigger sized modules when the user can actually see them